src/core/Element.cpp

```diff
@@ -1,7 +1,9 @@
 #include "Element.hpp"
 
 Element::Element(QString name, QString descr, int addr) :
```

Comment:
These parameters should be `const &`!

Reply:
QString has implicit move ctor and then it is the same in this case